tests/DfE.FindInformationAcademiesTrusts.Data.FiatDb.UnitTests/Repositories/ContactRepositoryTests.cs

```diff
@@ -1,102 +1,264 @@
-﻿using DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.UnitTests.Mocks;
-using DfE.FindInformationAcademiesTrusts.Data.Enums;
+﻿using DfE.FindInformationAcademiesTrusts.Data.Enums;
+using DfE.FindInformationAcademiesTrusts.Data.FiatDb.Models;
 using DfE.FindInformationAcademiesTrusts.Data.FiatDb.Repositories;
-using FluentAssertions;
-using Moq;
+using FluentAssertions.Execution;
 
 namespace DfE.FindInformationAcademiesTrusts.Data.FiatDb.UnitTests.Repositories;
 
-public class ContactRepositoryTests
+public class ContactRepositoryTests : BaseFiatDbTest
 {
     private readonly ContactRepository _sut;
-    private readonly MockFiatDbContext _mockFiatDbContext = new();
 
-    public ContactRepositoryTests()
+    public ContactRepositoryTests(FiatDbContainerFixture fiatDbContainerFixture) : base(fiatDbContainerFixture)
     {
-        _sut = new ContactRepository(_mockFiatDbContext.Object);
+        _sut = new ContactRepository(FiatDbContext);
     }
 
     [Fact]
-    public async Task
-        GetInternalContactsAsync_Should_Return_Valid_TrustRelationshipManager_WhenOneIsPresentForTheTrust()
+    public async Task GetInternalContactsAsync_should_return_nulls_when_there_are_no_contacts_for_trust()
     {
-        var trm = _mockFiatDbContext.CreateTrustRelationshipManager(1234, "Trust Relationship Manager",
-            "trm@testemail.com");
         var result = await _sut.GetInternalContactsAsync("1234");
-        result.TrustRelationshipManager.Should()
-            .BeEquivalentTo(trm, options => options.WithAutoConversion().ExcludingMissingMembers());
+
+        using (new AssertionScope())
+        {
+            result.TrustRelationshipManager.Should().BeNull();
+            result.SfsoLead.Should().BeNull();
+        }
     }
 
     [Fact]
-    public async Task GetInternalContactsAsync_Should_Return_Valid_SFSOLead_WhenOneIsPresentForTheTrust()
+    public async Task GetInternalContactsAsync_should_return_TrustRelationshipManager_when_there_is_no_SfsoLead()
     {
-        var sfsoLead = _mockFiatDbContext.CreateSfsoLead(1234, "SFSO Lead", "sfsolead@testemail.com");
+        FiatDbContext.Contacts.Add(new Contact
+        {
+            Email = "trm@testemail.com",
+            Name = "Trust Relationship Manager",
+            Role = ContactRole.TrustRelationshipManager,
+            Uid = 1234
+        });
+        await FiatDbContext.SaveChangesAsync();
+
         var result = await _sut.GetInternalContactsAsync("1234");
-        result.SfsoLead.Should()
-            .BeEquivalentTo(sfsoLead, options => options.WithAutoConversion().ExcludingMissingMembers());
+
+        using (new AssertionScope())
```

Comment:
TIL what AssertionsScopes were and they're pretty cool! (No need to change just thought it was neat)